### PR TITLE
Dami feature forum page

### PR DESCRIPTION
**Pull request by dakom0**
- Related Issue (include '#'): 569

- Description of changes made: Made last directory pages.

- Is the feature complete/bug resolved/etc..: Yes

- Any known bugs/strange behavior: My groups tab not routing to groups page.

- Is there specific feedback you would like on these changes: None

- Screenshot(s): 
Screen Shot 2020-05-18 at 9 54 08 AM

### CODE REVIEW

File: client/src/components/Card/Card.css [hunk 1]
```diff
@@ -115,7 +115,7 @@
 
 .Card__discover-link {
   text-align: center;
-  margin: auto 20px auto 20px;
+  margin: 50px 20px 50px 20px;
```
Comment (clarknoah) [inline client/src/components/Card/Card.css:118]: @dakom0 is changing the CSS going to have any unintended consequences for any other pages that are using this component?

File: client/src/components/InputTextField/InputTextField.css [hunk 2]
```diff
@@ -57,7 +57,7 @@
   -webkit-box-shadow: 0px 8px 5px -5px rgba(0, 0, 0, 0.53);
   -moz-box-shadow: 0px 8px 5px -5px rgba(0, 0, 0, 0.53);
   box-shadow: 0px 8px 5px -5px rgba(0, 0, 0, 0.53);
-  margin: 1rem 0 1rem 0;
+  margin: 1rem 0 1em 0;
```
Comment (clarknoah) [inline client/src/components/InputTextField/InputTextField.css:60]: @dakom0 same here

**Comment by clarknoah** [comment]
Screen Shot 2020-05-18 at 11 05 55 AM
I made a few comments, also, Per the figma, this should be a thicker purple, as well as flush with the banner. 

Also, the `My Friend` and `Groups` tab links are inoperable right now, please correct this!

**Comment by dakom0** [comment]
Yes. Youre right about the Card.css I forgot to undo the changes. The Input Text field is a very minimal change that increased the top margin. Not a very drastic change.

**Comment by dakom0** [comment]
I got the thicker border done. I don't have a space between the bar and banner though. Also I will work on the tabs to link to different pages.

**Pull request merged**